backend/account/authentication_controller.py

```diff
@@ -120,6 +120,8 @@ def user_organizations(self, request: Request) -> Any:
                     data={"domain": ex.data.get("domain"), "code": ex.code},
                 )
                 return response
+            Logger.error(f"Email address not valid: {ex}")
+            raise ex
```

Comment:
Ideally we should catch and handle the specific error. Here we generically catch all exceptions of which we assume that its an invalid email error if the condition has failed in line 114. Let's say we raise a different exception from the plugin, how will that get logged / handled?

Reply:
@chandrasekharan-zipstack added error codes and handled this.